milli/src/update/facets.rs

```diff
@@ -262,8 +262,8 @@ impl<'t, 'u, 'i> Facets<'t, 'u, 'i> {
 /// 1. a vector of grenad::Reader. The reader at index `i` corresponds to the elements of level `i + 1`
 /// that must be inserted into the database.
 /// 2. a roaring bitmap of all the document ids present in the database
-fn compute_facet_number_levels<'t>(
-    rtxn: &'t heed::RoTxn,
+fn compute_facet_number_levels(
+    rtxn: &'_ heed::RoTxn,
```

Comment:
Is this anonymous lifetime necessary?
```suggestion
    rtxn: &heed::RoTxn,
```

Reply:
This one is not needed.

Reply:
fixed